src/main/java/msa/innoit/prices/data/infrastructure/adapter/out/prices/repository/PriceRepository.java

```diff
@@ -1,7 +1,16 @@
 package msa.innoit.prices.data.infrastructure.adapter.out.prices.repository;
 
+import java.time.LocalDateTime;
 import msa.innoit.prices.data.infrastructure.adapter.out.prices.entity.PriceEntity;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
 
 public interface PriceRepository extends JpaRepository<PriceEntity, Integer> {
+
+  @Query(value = "SELECT * FROM T_PRICES WHERE PRI_PRODUCT_ID = ?2 AND "
```

Comment:
Implement JQL to separate database names dependencies for repository. Use this:

@Query(value = "SELECT p FROM PriceEntity p WHERE p.productId = ?2 AND "
      + "p.brandId = ?3 AND ?1 BETWEEN p.startDate AND p.endDate")

Reply:
solucionado